bibangamba/githubusers_react_native
# #167162366 build the login screen

#### What does this PR do?
adds a login screen to the application

#### Description of tasks to be done?
- build login component
- create GitHub login script to request token
- save the access token to async storage
- add unit test this functionality

#### How can this be manually tested?
- clone the repo and check out the branch
- run the application with `npm start`
- send the application to an emulator by typing `a`(android) or `i`(iOS)
- the application should open up on a login screen 
- entering good credentials will take you to a single screen with the GitHub logo

#### Screenshots
![demo gif](http://g.recordit.co/X2CIiKtZgM.gif)

#### What are the relevant pivotal tracker stories?
[#167162366](https://www.pivotaltracker.com/story/show/167162366)
